### DISCUSSION TITLE
Timeout on Alert Component

### DISCUSSION THREAD

**Discussion by NicoPowers** [cloudscape-design/components#1445] [Q&A] [answered]
Hi,

I noticed the Alert component is dismissible, but there is currently no way to have it automatically disappear, say, after 2 seconds.

What is the workaround to do this? Is there a component that would be better suited to implement something like that?

Thank you!

**Comment by just-boris** [comment, marked as answer]
Hello!

We have made a deliberate decision to not support auto-closing alerts. People read texts with different speed (especially if the text is not in their first language). To give everyone enough time to read the message, it is better to avoid auto-dismissing messages
